### PR TITLE
[SIS-025] Settings Page

### LINKED ISSUE #25: [SIS-025] Settings Page
See Figma

### PR DESCRIPTION
**Pull request by yuemily24**
# Description

- Add Settings page including all its buttons and pages
- Add `showBackButton` prop to `Header` and also clean up its icons

Ticket # 25

image
image
image

## Type of change

- [X] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [x] My PR title is formatted as [SIS-\<issue number\>] \<issue name\> (eg. [SIS-010] Create UI button)
- [x] I have linked my issue/ticket on the project board to this PR
- [x] I have sent my PR to #team-sistema-prs on Slack and requested two people to review
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if required)
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by harmitgoswami — APPROVED** [review]
LGTM! I have a comment though:

When ran this branch initially, I got a build error for "SistemaLogo.png doesn't exist" in src/components/Header.js. I'm not sure if this is only an error on my end, but since the .svg naming was changed to camelCase, this file doesn't exist anymore. If the file is still there on your end, please delete it

otherwise everything looks perfect!

**Comment by ramyzhang** [comment]
RT with Harmit on the .svg, but otherwise it also looks good on iOS!!!
also wraow code so clean

File: src/home/ExpandedCard.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ import {
   Text,
   ScrollView,
   Dimensions,
-  TouchableOpacity,
 } from 'react-native';
 import AddButton from './components/AddToLessonButton';
 import BackArrow from '../../assets/backArrow';
```
Comment (ramyzhang) [inline src/home/ExpandedCard.js]: not using this anymore so can delete BackArrow from here!

**Review by ramyzhang — CHANGES_REQUESTED** [review]

File: src/Components/Header.js [hunk 2]
```diff
@@ -5,47 +5,83 @@ import {
   View,
   Text,
   TouchableOpacity,
+  ScrollView,
 } from 'react-native';
 import SistemaButton from '../Components/SistemaButton';
-import SistemaLogo from '../../assets/sistemaLogo.svg';
+import SistemaLogo from '../../assets/SistemaLogo.svg';
```
Comment (ramyzhang) [inline src/Components/Header.js]: change back to sistemaLogo.svg
i think maybe pulling from master doesn't cut it bc of git's weirdness, so you might have to just manually change it locally

**Review by ramyzhang — CHANGES_REQUESTED** [review]

**Pull request merged**